src/Components/WebAssembly/Server/src/DebugProxyLauncher.cs

```diff
@@ -51,7 +51,7 @@ private static async Task<string> LaunchAndGetUrl(IServiceProvider serviceProvid
             var processStartInfo = new ProcessStartInfo
             {
                 FileName = muxerPath,
-                Arguments = $"exec \"{executablePath}\" --owner-pid {ownerPid} --DevToolsUrl {devToolsHost}",
+                Arguments = $"exec \"{executablePath}\" --OwnerPid {ownerPid} --DevToolsUrl {devToolsHost}",
```

Comment:
This isn't strictly related to lazy-loading but we need this to consume https://github.com/dotnet/runtime/pull/42551. Doing it now to avoid a double PR later.